packages/components/pie-tag/src/tag.scss

```diff
@@ -1 +1,152 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+// Base tag styles
+.c-tag {
+    // Custom Property Declarations
+    // These are defined here instead of :host to encapsulate them inside Shadow DOM
+    --tag-font-family: var(--dt-font-body-s-family);
+    --tag-font-weight: var(--dt-font-body-s-weight);
+
+    // The following values set to default background and color
+    // currently this sets the neutral large tag styles
+    --tag-bg-color: var(--dt-color-container-inverse);
+    --tag-color: var(--dt-color-content-inverse);
+
+    // transparent to variable to function properly in component tests
+    --tag-transparent-bg-color: transparent;
+
+    // Heights for the different tag sizes
+    --tag-height-large: 24px;
+    --tag-height-small: 16px;
+
+    // Pie Webc Icon var that is used to ensure the correctly sized icon passed in a slot
+    --icon-display-override: block;
+    --icon-size-override: 16px;
+
+    display: inline-flex;
+    vertical-align: middle;
+    align-items: center;
+    justify-content: center;
+    gap: var(--dt-spacing-a);
+    height: var(--tag-height);
+    padding: var(--tag-padding);
```

Comment:
I'd setup the default value of this above in your variable declarations (set to whatever the default size/variation is) – just makes it a bit easier to see whats being set as variables altogether.

Same for height/font-size etc.